.github/workflows/run_periodic_tests.yml

```
@@ -24,22 +24,96 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  style:
+  run_unit_tests:
+    runs-on: ${{ matrix.os }}
```

```suggestion
  run_unit_tests:
    name: Unit tests (${{ matrix.os }} / Python ${{ matrix.python-version }})
    runs-on: ${{ matrix.os }}
```

We can make the names of the jobs resemble those for other ones, similar comment below for the `integration` job